java/ql/lib/semmle/code/java/Member.qll

```diff
@@ -306,7 +306,9 @@ class Callable extends StmtParent, Member, @callable {
    */
   Callable getKotlinParameterDefaultsProxy() {
```

Comment:
I hope this predicate is well-tested. I'll tentatively say it looks plausible, but it sure is involved.